src/OpenLoco/src/Ui/Windows/Construction/ConstructionTab.cpp

```diff
@@ -54,6 +54,11 @@ namespace OpenLoco::Ui::Windows::Construction::Construction
     static loco_global<uint8_t, 0x0112C2E9> _alternateTrackObjectId; // set from GameCommands::createRoad
     static loco_global<uint8_t[18], 0x0050A006> _availableObjects;   // top toolbar
 
+    // TODO: move to ConstructionState when no longer a loco_global?
```

Comment:
What is this comment saying? Which loco_global?

Reply:
I see, the comment isn't really that clear to the reader but once you know it makes sense I guess, anyway leave it as is.